README.md

```diff
@@ -25,7 +25,7 @@ We have a public calendar that lists Data Together events and a Youtube playlist
 📅 [**Data Together Calendar**](https://calendar.google.com/calendar/embed?src=u75o4fbnv59006peo07nv67vsg%40group.calendar.google.com&ctz=America%2FToronto)  
 ▶️ [**Call Playlist**](https://www.youtube.com/playlist?list=PLtsP3g9LafVul1gCctMYGm9sz5FUWr5bu)
 
-### Weekly Roundtables
+### ~~Weekly Roundtables~~
```

Comment:
Would it help to explicitly say that these are on hold?

Reply:
Oh, I missed that. Let's merge it 👍

Reply:
Ehhh, thinking more about this I think you are right @lightandluck and we want to be more current-- I think I read too quickly before, as this seems to imply that we might be returning to a regular schedule.

Reply:
Updated above @lightandluck ! What do you think?

Reply:
Looks great! Thanks for updating all of this 🙏